ciao-controller/internal/datastore/sqlite3db.go

```diff
@@ -838,14 +838,16 @@ func (ds *sqliteDB) getWorkloadDefaults(ID string) ([]payloads.RequestedResource
 func (ds *sqliteDB) getWorkloadStorage(ID string) (*types.StorageResource, error) {
 	query := `SELECT volume_id, bootable, persistent, size,
 			 source_type, source_id
-		  FROM 	workload_resources
+		  FROM 	workload_storage
```

Comment:
Could you squash this patch into the first one in the series where this function is added?